.github/workflows/publish.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout
-      uses: actions/checkout@v1
+      uses: actions/checkout@v3
```

Comment:
Does this checkout all the tags too?

The next line is `git describe --tags --exact-match` so that has to work

Reply:
I see. I've added a [fetch depth](https://github.com/actions/checkout/tree/v3#fetch-all-history-for-all-tags-and-branches),